fearless/Common/View/IconTitleValueView.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+import SoraUI
+
+class IconTitleValueView: UIView {
+    let imageView = UIImageView()
+
+    let titleLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = R.color.colorLightGray()
+        label.font = UIFont.p1Paragraph
+        return label
+    }()
+
+    let valueLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = R.color.colorWhite()
+        label.font = UIFont.p1Paragraph
+        return label
+    }()
+
+    let borderView = UIFactory.default.createBorderedContainerView()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupLayout()
+    }
+
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupLayout() {
+        addSubview(borderView)
+        borderView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+
+        addSubview(imageView)
+        imageView.snp.makeConstraints { make in
+            make.leading.centerY.equalToSuperview()
+            make.size.equalTo(16.0)
+        }
+
+        addSubview(titleLabel)
+        titleLabel.snp.makeConstraints { make in
+            make.leading.equalTo(imageView.snp.trailing).offset(9.0)
+            make.centerY.equalToSuperview()
+        }
+
+        addSubview(valueLabel)
+        valueLabel.snp.makeConstraints { make in
+            make.trailing.centerY.equalToSuperview()
+            make.leading.greaterThanOrEqualTo(titleLabel.snp.trailing).offset(8.0)
```

Comment:
I would add compressionResistancePriority for situations when neighbour content is too large

Reply:
I think in this case it will apply the same priority implicitly